administrator/components/com_associations/models/forms/filter_associations.xml

```diff
@@ -52,6 +51,8 @@
 				>
 				JASSOCIATIONS_DESC
 			</option>
+			<option value="a.menutype ASC">COM_MENUS_HEADING_MENU_ASC</option>
+			<option value="a.menutype DESC">COM_MENUS_HEADING_MENU_DESC</option>
```

Comment:
Maybe we can add this (and others if needed) options dinamicly as we do for the column ?